geode-core/src/main/java/org/apache/geode/cache/RegionFactory.java

```diff
@@ -100,6 +100,16 @@ protected RegionFactory(InternalCache cache, RegionAttributes ra) {
     this.attrsFactory = new AttributesFactory<K, V>(ra);
   }
 
+  /**
+   * For internal use only.
+   *
+   * @since GemFire 9.10
+   */
```

Comment:
Found a need to add a new API so that I could pass and modify a regionFactory.

Reply:
If it's on the public API, I don't think it should be marked for internal use.

Reply:
Good point...